packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -586,6 +578,22 @@ export function disposeAllViewsFromCall(
       }
     }
   }
+  /* @conditional-compile-remove(together-mode) */
+  const callFeatureStreams = internalContext.getCallFeatureRenderInfosForCall(callId);
+  /* @conditional-compile-remove(together-mode) */
+  if (callFeatureStreams) {
+    for (const [, featureStreams] of callFeatureStreams.entries()) {
```

Comment:
nit: you can use `.values()` instead of `.entries()` if you don't care about the keys